app/views/oidc/api_key_roles/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<% @title = t(".api_key_role_name", name: @api_key_role.name) %>
+
+<div class="t-body">
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:token) %></h3>
+  <div class="push--s">
+    <code><%= @api_key_role.token %></code>
+  </div>
+
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:provider) %></h3>
+  <div class="push--s">
+    <%= link_to @api_key_role.provider.issuer, @api_key_role.provider.issuer %>
+  </div>
+
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:api_key_permissions) %></h3>
+  <div class="push--s">
+  <code><pre><%= JSON.pretty_generate @api_key_role.api_key_permissions.as_json %></pre></code>
+  </div>
```

Comment:
Showing the raw json is a bit hard to read. Why not use the scopes, valid_for and gems from `OIDC::ApiKeyPermissions`?